### ISSUE TITLE
Plugging in VR headset after page load not detected

### ISSUE THREAD

**Issue by leweaver** [aframevr/aframe#3019]
**Description:**
When plugging in an HMD after page load, clicking the Enter VR button still takes you to the 2D mouse driven mode, rather than entering VR.

This appears to be related to ([three.js #12109](https://github.com/mrdoob/three.js/issues/12109))
A workaround could be to re-implement the VRControls class in a-frame; it is only an example in three.js after all and is quite a small class to begin with.

- A-Frame Version: master
- Platform / Device: Windows Mixed Reality
- Reproducible Code Snippet or URL: https://aframe.io/examples/showcase/helloworld/

**Comment by dmarcos** [comment]
Am I correct to assume that this is happening in Edge?

**Comment by leweaver** [comment]
Sorry I omitted that. I have tested the following combinations, it reproduces on all of them:

- Windows Mixed Reality & Microsoft Edge
- Vive & Firefox
- Vive & Chromium

Test steps were:

1. Ensure device is disconnected, and that VR software is shut down
2. Open browser, navigate to https://aframe.io/examples/showcase/helloworld/
3. Connect hardware, wait for the home screen to be visible in the HMD
4. Press Enter VR on the a-frame sample
5. It will enter full screen mode, rather than VR.
